src/modules/personenkontext/api/dbiam-personenkontext-workflow.controller.ts

```diff
@@ -243,23 +246,47 @@ export class DbiamPersonenkontextWorkflowController {
             params.personalnummer || undefined,
             params.befristung || undefined,
         );
+        if (savedPersonWithPersonenkontext instanceof Error) {
+            this.logger.error(
```

Comment:
`instanceof Error` can have very weird behaviour in JS. Can we switch this to `!(savedPersonWithPersonenkontext instanceof PersonPersonenkontext)`?

Reply:
Wont work this way.
"The instanceof operator expects the right-hand operand to be a class or a constructor function, but here PersonPersonenkontext is likely a TypeScript interface or type alias, which is purely a compile-time construct and not available at runtime."

Reply:
Ah, I thought it already was a class. In that case keep it like this, although I don't like how were returning multiple different things here (This should really be a `Result<>` instead, but it was already like this)